src/geometries/CapsuleGeometry.js

```diff
@@ -0,0 +1,39 @@
+import { Path } from '../extras/core/Path';
+import { LatheGeometry } from './LatheGeometry';
+
+class CapsuleGeometry extends LatheGeometry {
+
+	constructor( radius = 1, length = 1, capSegments = 4, radialSegments = 8 ) {
+
+		const R1 = 0;
+		// max here prevents failing when height is 0
+		const a = Math.asin( R1 / Math.max( Number.EPSILON, length ) );
+
+		const path = new Path();
+		path.absarc( 0, 0, radius, Math.PI * 1.5, a );
+		path.absarc( 0, length, radius, a, Math.PI * 0.5 );
```

Comment:
Since there's no diff between the two radii, this can be simplified like this

```suggestion
		const path = new Path();
		path.absarc( 0, 0, radius, Math.PI * 1.5, 0 );
		path.absarc( 0, length, radius, 0, Math.PI * 0.5 );
```

yay or nay?

Reply:
Simpler is always better 🤓